app/src/main/java/fr/neamar/kiss/dataprovider/SearchProvider.java

```diff
@@ -10,13 +11,17 @@
 import java.util.Map;
 import java.util.Set;
 import java.util.TreeSet;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 
 import fr.neamar.kiss.loader.LoadSearchPojos;
 import fr.neamar.kiss.pojo.Pojo;
 import fr.neamar.kiss.pojo.SearchPojo;
 
 public class SearchProvider extends Provider<SearchPojo> {
     private SharedPreferences prefs;
+    public static final String URL_REGEX = "^((https?|ftp)://|(www|ftp)\\.)?[a-z0-9-]+(\\.[a-z0-9-]+)+([/?].*)?$";
```

Comment:
I tried adding the regex you proposed but it seems like there's something wrong. As I said I'm not really good with these so I don't really know what's wrong (I mean I understand the error but am not really sure how to fix it without braking the expression)
Here's a screenshot:
![schermata 2016-11-30 alle 17 38 27](https://cloud.githubusercontent.com/assets/19847464/20761632/e26f9e60-b723-11e6-95fb-ecb53ac2b9c5.png)